src/libs/relay/conduit_relay_io_identify_protocol.cpp

```diff
@@ -159,12 +193,14 @@ identify_file_type(const std::string &path,
                 if(test_str.find("{") != std::string::npos)
                 {
                    file_type = "json";
```

Comment:
we still have this issue no? #1291

Reply:
looks good

Reply:
although it is still strange to have
```
if (yaml case)
   yaml
if (json case)
   json
```

couldn't it be
```
if (yaml case)
   yaml
else if (json case)
   json
```
instead?

Reply:
that will not work b/c json often includes `:` which is what we look for for yaml.

Reply:
if it possible to switch the order? if json then json else if yaml then yaml

Reply:
if not then I will add a comment so this doesn't confuse anyone in the future